changelog.d/10672.misc

```diff
@@ -0,0 +1 @@
+Update our periodic test of Synapse against Twisted trunk to avoid opening duplicate issues.
```

Comment:
probably just set this to the same as #10651, rather than having two separate changelog entries?

Reply:
Ah right, I keep forgetting about how Towncrier does that